lib/hacluster.pm

```diff
@@ -228,8 +229,39 @@ sub ha_export_logs {
 
 sub check_cluster_state {
     assert_script_run "$crm_mon_cmd";
+    assert_script_run "$crm_mon_cmd | grep -i 'no inactive resources'";
     assert_script_run 'crm_mon -1 | grep \'partition with quorum\'';
     assert_script_run 'crm_mon -s | grep "$(crm node list | wc -l) nodes online"';
+    assert_script_run 'crm_verify -LV';
+}
+
+# Wait for resources to be started
+sub wait_until_resources_started {
+    my @cmds    = ('crm cluster wait_for_startup', "$crm_mon_cmd | grep -i 'no inactive resources'");
+    my $timeout = 120;
+    my $ret     = undef;
+
+    # Execute each comnmand to validate that the cluster is running
```

Comment:
Ouch ! I just saw a mistake, 'comnmand' instead of 'command'...

As the PR is merged I will try to correct this in a next PR :)

Reply:
True, I guess we need built -in spell checker. @okurz is really good in catching those =)